api/api-agents-v4/src/main/java/com/thoughtworks/go/apiv4/agents/AgentsControllerV4.java

```diff
@@ -80,8 +80,8 @@ public String controllerBasePath() {
     @Override
     public void setupRoutes() {
         path(controllerBasePath(), () -> {
-            before("", this::setContentType);
-            before("/*", this::setContentType);
+            before("", mimeType, this::setContentType);
+            before("/*", mimeType, this::setContentType);
```

Comment:
I just realized that this is a copy-paste error. All controllers use this particular method, without the `mimeType` param.